Babel support and Databse Contract Test

yiran29 opened the pull request:
*Description of changes:*
1) AWS Remote DB User support
2) Contract Tests for MySQL and PostgreSQL
Currently have to comment out RemoteOperation field test, the issue is described in https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81

*Test Summary:* 
**PostgreSQL**
`===== 4 passed in 25.86s =====`

**MySQL**
`====== 4 passed in 42.45s ======`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

vastin commented on test/contract-tests/tests/test/amazon/efcore/efcore_test.py:
Why is `local_operation` validation removed?

vastin commented on test/contract-tests/tests/test/amazon/netcore/netcore_test.py:
Why are `request_method` and `local_operation` validations removed?

vastin commented:
There is merge conflict. Please rebase the pull request.

vastin commented:
The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?

yiran29 commented:
> The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?

The time is mainly from the Auto Instrumentation Initialization

vastin commented:
> > The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?
> 
> The time is mainly from the Auto Instrumentation Initialization

Please add following in Dockerfile and it will eliminate the delay.
```
ENV RESOURCE_DETECTORS_ENABLED=false
```

yiran29 commented:
> > > The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?
> > 
> > 
> > The time is mainly from the Auto Instrumentation Initialization
> 
> Please add following in Dockerfile and it will eliminate the delay.
> 
> ```
> ENV RESOURCE_DETECTORS_ENABLED=false
> ```

Updated the config and rerun the tests. Updated test summary.

vastin commented on test/contract-tests/tests/test/amazon/mysql/mysql_test.py:
```diff
@@ -0,0 +1,184 @@
+        attributes_dict: Dict[str, AnyValue] = self._get_attributes_dict(attributes_list)
+        self._assert_str_attribute(attributes_dict, AWS_LOCAL_SERVICE, self.get_application_otel_service_name())
+        self._assert_str_attribute(attributes_dict, AWS_REMOTE_SERVICE, self.get_remote_service())
+        # TODO: https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81
```
In [issue #81](https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81), there is `db.statement: SELECT * FROM Blogs`. Why is AWSRemoteOperation set to unknown?

yiran29 replied:
The issue happened here: https://github.com/aws-observability/aws-otel-dotnet-instrumentation/blob/main/src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsMetricAttributeGenerator.cs#L489
Briefly tested without regex test will show the remote operation

vastin reviewed (approved):
LGTM
